cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
What's going on here? Undo your changes- but why is this file there to start with?